docker-services.yml

```diff
@@ -29,7 +29,7 @@ services:
       - "15672:15672"
       - "5672:5672"
   es:
-    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.9.1
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.1.1
```

Comment:
why did you downgrade elasticsearch?

Reply:
to have all the environments with the same version of elasticsearch, which caused that we missed the fact the boosting query will not work on sandbox